### PR TITLE
Rebased changes and additional fixes.

### PR DESCRIPTION
**Pull request by JuviAndaya**
Hi Tony, 

I rebased the vm code version to the wessex master. I added some fixes too for the Supply page. 

Regards, 

Juvi

### CODE REVIEW

File: chellow/templates/base.html [hunk 1]
```diff
@@ -126,8 +126,7 @@
     		<link rel="stylesheet" href="/style" />
 		    <link rel="stylesheet" href="/static/css/bootstrap.min.css" />
         	<link rel="stylesheet" href="/static/font-awesome-4.6.3/css/font-awesome.min.css" />
-        	<link rel="stylesheet" href="/chellowcss" />
-        	
+        	<link rel="stylesheet" href="{{context_path}}/chellowcss" />
```
Comment (tlocke) [inline chellow/templates/base.html]: There's a {{context_path}} here, but we don't use it any more. Also, is there any reason why the stylesheet can't be in 'static'?

File: chellow/templates/hhdc_contract_edit.html [hunk 2]
```diff
@@ -151,7 +151,6 @@
 	    <!-- End Section Content -->
 	</div>
 	<!-- End Layer Section -->
->>>>>>> Update Contracts Module design
```
Comment (tlocke) [inline chellow/templates/hhdc_contract_edit.html]: This looks like a rebase marker that needs to be removed.

File: chellow/templates/site_edit.html [hunk 3]
```diff
@@ -33,13 +33,8 @@
 					<form method="post" action="">
 						<fieldset>
```
Comment (tlocke) [inline chellow/templates/site_edit.html]: There's an open <fieldset> but I can't see an end tag.

File: chellow/templates/site_edit.html [hunk 4]
```diff
@@ -33,13 +33,8 @@
...
 							<input type="hidden" name="site_id" value="{{site.id}}">
-							<!-- <legend>
-								Are you sure you want to delete this site and any associated snags?
-							</legend> -->
 							<input type="submit" name="delete" value="delete"> 
-							<input type="reset" value="Cancel" onClick="window.location='/sites/{{site.id}}/edit'" />
-							
-						</fieldset>
+							<input type="reset" value="Cancel" onClick="window.location='{{context_path}}/sites/{{site.id}}/edit'" />													</fieldset>
```
Comment (tlocke) [inline chellow/templates/site_edit.html]: Probably not a good idea to use JavaScript for just the cancel button. Better to do with HTML / CSS.

File: chellow/templates/sites.html [hunk 5]
```diff
@@ -18,8 +18,8 @@
 		<div class="section-data">
 			<div class="standardForm textWidthAuto">
 				<div class="row">
-					<div class="col-sm-6">	
-						<form>
+					<div class="col-sm-6">
+						<form action="">
 							<fieldset>
```
Comment (tlocke) [inline chellow/templates/sites.html]: The 'action' attribute shouldn't be empty. See the HTML5 spec:

https://www.w3.org/TR/html5/forms.html#attr-fs-action

File: chellow/views.py [hunk 6]
```diff
@@ -269,9 +269,11 @@ def style_get():
 
 @app.route('/chellowcss', methods=['GET'])
 def chellowcss_get():
+    props = Contract.get_non_core_by_name(g.sess, 'configuration'). \
+        make_properties()
     response = make_response(
         render_template(
-            'css/chellow.css'))
+            'css/chellow.css', background_colour=props['background_colour']))
```
Comment (tlocke) [inline chellow/views.py]: What's the difference between /style and /chellowcss? Do we need both?

**Comment by JuviAndaya** [comment]
Hi @tlocke , I rebased again.Let me know if i missed something.

Regarsds,

**Comment by tlocke** [comment]
Hi @JuviAndaya, I think the main things are:

* We seem to have two non-static stylesheets, /style and /chellowcss. Is there are reason for this, or can one of them be static?
* The 'action' attribute of forms shouldn't be empty. See the HTML5 spec: https://www.w3.org/TR/html5/forms.html#attr-fs-action

Thanks,

Tony.

**Comment by tlocke** [comment]
Looking at the change for `chellow/templates/channel_snags.html`, there still seems to be an empty `action` attribute in the `form` element.

**Comment by JuviAndaya** [comment]
Hi Tony,

Here's the updated code.

File: test/imprimatur.py [hunk 7]
```diff
@@ -6517,12 +6517,11 @@ def virtual_bill(supply_source):
         'path': '/hhdc_contracts/34/auto_importer',
         'tries': {},
         'regexes': [
-            r"Finished loading",
```
Comment (tlocke) [inline test/imprimatur.py]: Why is this line removed?

File: test/imprimatur.py [hunk 8]
```diff
@@ -6517,12 +6517,11 @@ def virtual_bill(supply_source):
...
             r'<a href="/hhdc_contracts/34/auto_importer">Refresh page</a>']},
     {
         'path': '/hhdc_contracts/34',
         'regexes': [
-            r"hh_data\.df2"]},
```
Comment (tlocke) [inline test/imprimatur.py]: Why is this bit removed?

File: test/imprimatur.py (hunk 7 above)
Comment (JuviAndaya) [inline test/imprimatur.py]: the test scripts failed. I compared it with master and there's no finished loading in the auto importer.

File: chellow/templates/hhdc_contract_edit.html (hunk 2 above)
Comment (tlocke) [inline chellow/templates/hhdc_contract_edit.html]: Looks like the rebase marker is still there...
